projects/rocprofiler-sdk/source/lib/python/rocpd/csv.py

```diff
@@ -51,11 +455,24 @@ def execute(input, config=None, window_args=None, **kwargs):
 def add_args(parser):
     """Add csv arguments."""
 
-    return []
+    csv_options = parser.add_argument_group("CSV options")
+    csv_options.add_argument(
+        "--titled-columns",
+        action="store_true",
+        default=False,
```

Comment:
```suggestion
        default=True,
```
I think the default should have Titled Columns = True, don't you think so?  Doesn't that match the current output files?

I'm not convinced we need this option for CSVs.  Why would anyone want to turn it off :)

Jonathan's suggestion for "titled_columns" was for the query module.  I don't think CSV module really needs it.

Reply:
Removed the titled_columns argument.